department-of-veterans-affairs/vets-website
# 10 7959f 1 intro confirmation

## Summary

Updates form 10-7959f-1 to have form-specific introduction and confirmation pages

- Updating these files caused some errors in other files that were fixed. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76792

## Testing done

- Manual Testing
- Ran yarn test:coverage-app ivc/champva/10-7959f-1 to verify existing tests run and pass

## Screenshots
Top of intro page (it's too long for one screenshot)
![Screenshot 2024-02-21 at 2 29 56 PM (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/20195737/4984e18e-72c1-4a96-ad0d-d39573cd553a)
Bottom of intro page
![Screenshot 2024-02-21 at 2 30 39 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/20195737/a4f710ac-76e6-469d-ae08-d2d03a354898)
Confirmation page
![Screenshot 2024-02-21 at 2 29 31 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/20195737/6ea0203d-b024-4779-ae85-7e39a37b264b)

## What areas of the site does it impact?

This form only

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback
N/A
